includes/class-llms-loader.php

```diff
@@ -132,6 +132,7 @@ public function includes() {
 
 		// Classes.
 		require_once LLMS_PLUGIN_DIR . 'includes/class-llms-assets.php';
+		require_once LLMS_PLUGIN_DIR . 'includes/class-llms-dom-document.php';
```

Comment:
Method needs `@since` tag and the class need's its `@version` tag updated

Reply:
done in 47d61de